### ISSUE TITLE
Support Zeroize

### ISSUE THREAD

**Issue by afck** [zkcrypto/pairing#106]
If `Fr` implemented [`Zeroize`](https://docs.rs/zeroize/1.0.0/zeroize/trait.Zeroize.html), it would be easier to make sure all values get cleared from memory after a computation involving secrets, e.g. BLS keys.

**Comment by str4d** [comment]
We are in the process of removing the BLS12-381 implementation from `pairing`, and replacing it with the `bls12-381` crate. We should look at making this change there, and then once the refactor is done we can see whether there are any pairing-friendly traits in `pairing` that would benefit from having a `Zeroize` type constraint.

**Comment by afck** [comment]
Closing in favor of https://github.com/zkcrypto/bls12_381/issues/14.
